src/test/resources/feature/2-mittente/ente_radice_e_ente_figlio.feature

```diff
@@ -95,7 +95,55 @@ Feature: Ente figlio e Ente radice
     And Si verifica che Api Key sono diversi
     Then Nel pop up visualizza cliccare sul tasto chiudi
     And Si clicca sul bottone esci
+    And Logout da portale mittente
+
+
+
+  @test5
+  Scenario: PN-10419 - Ente Figlio - Verifica presenza notifiche da parte del delegato
+    Given Login Page mittente viene visualizzata
+      | url | https://selfcare.dev.notifichedigitali.it |
+    When Login con mittente
+      | user   | ggiorgi |
+      | pwd    | test    |
+      | comune | Viggiu  |
+    And Si clicca sul bottone test
+    And Si clicca bottone accetta cookies
+    And Si sceglie ente figlio "EDILIZIA PRIVATA E SUAP"
+    And Nella pagina Piattaforma Notifiche cliccare sul bottone Invia una nuova notifica
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Informazioni preliminari
+    Then Nella section Informazioni preliminari si inseriscono i dati della notifica senza gruppo
+      | oggettoNotifica   | Pagamento RATA IMU |
```

Comment:
use send notification with api

Reply:
for this mittente this approach is good. And no need for payment in the notification

Reply:
It doesn't matter if there is payment